aiida/common/escaping.py

```diff
@@ -35,6 +35,7 @@ def escape_for_bash(str_to_escape):
     within triple quotes to make it work, getting finally: the complicated
     string found below.
     """
+    str_to_escape = str(str_to_escape)
```

Comment:
if `str_to_escape` is `None` this will now return `'None'` i.e. the literal string 'None' which is not what we want. This should at the very least be moved after the conditional that checks for `None`

Reply:
Yeah, that would be great, its definitely a bit more subtle an issue than I thought it was...